docker/pythonpath_dev/superset_config.py

```diff
@@ -108,6 +108,8 @@ class CeleryConfig(object):
 
 SQLLAB_CTAS_NO_LIMIT = True
 
+MAPBOX_API_KEY = os.getenv("MAPBOX_API_KEY")
+
```

Comment:
We appear to be using `get_env_variable` in this file, maybe use that here, too, for consistency?
```suggestion
MAPBOX_API_KEY = get_env_variable("MAPBOX_API_KEY")

```